src/cljs/remote_manager/controller.cljs

```diff
@@ -103,11 +103,11 @@
 
 (defn disconnect!
   []
+  (swap! store/state assoc {:connected? false})
```

Comment:
Fixed

You're totally right. That's what I get for copy/pasting :stuck_out_tongue: